addons/a11y/src/a11yHighlight.ts

```diff
@@ -21,9 +21,12 @@ const highlight = (infos: HighlightInfo) => {
   const id = HIGHLIGHT_STYLE_ID;
   resetHighlight();
 
+  // Make sure there are no duplicated selectors
+  const elements = Array.from(new Set(infos.elements));
```

Comment:
I noticed that sometimes the elements array would be of size 20, but actually 10 of these selectors would be just duplicated, resulting in a longer stylesheet than necessary